### ISSUE TITLE
HttpResponse.ofRedirect()

### ISSUE THREAD

**Issue by trustin** [line/armeria#914]
Related: https://github.com/line/centraldogma/pull/33 #773 #774 

It would be useful when a user wants to send a redirect response:

```java
HttpResponse.ofRedirect("/new-location");
HttpResponse.ofRedirect("/new-location/%s", someVar);
HttpResponse.ofRedirect(RedirectType.PERMANENT, "/new-location");
HttpResponse.ofRedirect(RedirectType.PERMANENT, "/new-location/%s", someVar");
```

/cc @hyangtack @abdulazizali77

**Comment by abdulazizali77** [comment]
@trustin i will start on this :)

**Comment by trustin** [comment]
@abdulazizali77 Awesome. :-D

**Comment by cnabro** [comment]
@trustin @abdulazizali77 
Is there any progress this issue?
I need this. can I implement this spec?
now, I handle it like this..

It's not uncomfortable, but it would be nice to have it.
```kotlin
return HttpResponse.of(
                    ResponseHeaders.of(
                            HttpStatus.MOVED_PERMANENTLY,
                            HttpHeaderNames.LOCATION,
                            "https://github.com/line/armeria/issues/914"
                    )
            )
```

**Comment by abdulazizali77** [comment]
@cnabro i think this is already merged ....?

@trustin ! apologies for never following up on other things! also i never thanked you for the 'gift'! it was so long ago! thank you thank you! i hope you and all linecorp are well!

**Comment by cnabro** [comment]
@abdulazizali77 
You mean this issue https://github.com/line/armeria/issues/773?
I think this request is closed because of this issue https://github.com/line/armeria/issues/850.
If I was wrong. please tell about this. Thanks.

**Comment by ikhoon** [comment]
Closed via #3059
